### ISSUE TITLE
Using SecureString for strPass in WMIConnect

### ISSUE THREAD

**Issue by athiasjerome** [nccgroup/WMIcmd#1]
Suggested:
public void WMIConnect(string strDomain, string strUser, SecureString strPass)

**Comment by olliencc** [comment]
So the process is only relatively short lived. What risk do you foresee as-is?

**Comment by kempy007** [comment]
Perhaps also allow interactive input of the password and masque the input. 
The risk would be passwords stored in process monitoring logs such as Sysinternals Sysmon, Carbon Black and any other EDR tools. Fab tool by the way :+1:

**Comment by olliencc** [comment]
@kempy007 yep not a problem.. will look to work on this ideally this week.

**Comment by olliencc** [comment]
initial commit for this is now in 87e60353bfa890bebb8c19839c8487b88f27cde9

**Comment by kempy007** [comment]
Have tested, but -a option always results in failed login and account lockout.
[!] Failed to connect Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
